src/main/preload.js

```diff
@@ -0,0 +1,22 @@
+const { contextBridge, ipcRenderer } = require('electron');
+
+contextBridge.exposeInMainWorld('electron', {
+  ipcRenderer: {
+    send: ipcRenderer.send,
+    sendSync: ipcRenderer.sendSync,
```

Comment:
Exposing ipcRenderer.send and ipcRenderer.sendSync goes against the security considerations of context isolation.

https://www.electronjs.org/docs/tutorial/context-isolation#security-considerations
> This would allow any website to send arbitrary IPC messages which you do not want to be possible. The correct way to expose IPC-based APIs would instead be to provide one method per IPC message.

Reply:
Fixed, thanks for the callout!